librz/core/cmd_descs/cmd_open.yaml

```diff
@@ -47,28 +66,56 @@ commands:
       - RZ_OUTPUT_MODE_QUIET
       - RZ_OUTPUT_MODE_JSON
       - RZ_OUTPUT_MODE_TABLE
+    details:
+      - name: Examples
+        entries:
+          - text: "ol.j"
+            comment: Show currently opened file in JSON mode
   - name: o-
     cname: open_close
     summary: Close file descriptor
     args:
       - name: fd
         type: RZ_CMD_ARG_TYPE_NUM
+    details:
+      - name: Examples
+        entries:
+          - text: "o-"
+            arg_str: " 3"
+            comment: Close the file with file descriptor 3
   - name: o--
     cname: open_close_all
     summary: Close all files
     args: []
+    details:
+      - name: Examples
+        entries:
+          - text: "o--"
+            comment: Close all files
   - name: oc
     cname: open_core_file
     summary: Close all opened files and open <file>, like relaunching rizin
     args:
       - name: file
         type: RZ_CMD_ARG_TYPE_FILE
+    details:
+      - name: Examples
+        entries:
+          - text: "oc"
+            arg_str: " myfile.exe"
+            comment: Close all opened files and open myfile.exe in read-only mode
```

Comment:
Again, i think it makes sense to add the Example section only if/when it adds useful information, otherwise we can just leave the summary as it is.